package.json

```diff
@@ -42,7 +41,8 @@
   },
   "peerDependencies": {
     "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react-dom": "^0.14.0",
+    "react-tap-event-plugin": "^0.2.0"
```

Comment:
@oliviertassinari Gotcha, but will leave `react-tap-event-plugin` in `dependencies` will cause npm  to install `history` package in the `node_modules` instead of `node_modules/react-tap-event-plugin/node_modules/`? 
That acts like we have a `history` package in `dependencies`. Haven't figure out the reason.